templates/flavors/README.md

```diff
@@ -14,3 +14,77 @@ appended to the end of the cluster-template.yaml, e.g cluster-template-{director
 flavor can be used by specifying `--flavor {directory-name}`.
 
 To generate all CAPZ flavors, run `make generate-flavors`.
+
+
+## Running flavor clusters as a tilt resource
+
+#### From CLI
+run ```tilt up ${flavors}``` to spin up worker clusters in azure represented by a tilt local resource.  You can also modify flavors while tilt is up by running ```tilt args ${flavors}```
+
+#### From Tilt Config
+Add your desired flavors to tilt_config.json:
+```json
+{
+    "worker-flavors": ["default", "aks external-cloud-provider", "machinepool", "system-assigned-identity", "user-assigned-identity"]
+}
+```
+
+#### Requirements
+Please note your tilt-settings.json must contain at minimum the following fields:
+```json
+{
+    "kustomize_substitutions": {
+        "AZURE_SUBSCRIPTION_ID_B64": "******",
+        "AZURE_TENANT_ID_B64": "******",
+        "AZURE_CLIENT_SECRET_B64": "******",
+        "AZURE_CLIENT_ID_B64": "******",
+        "AZURE_SUBSCRIPTION_ID": "******",
+        "AZURE_TENANT_ID": "******",
+        "AZURE_CLIENT_ID": "******",
+        "AZURE_CLIENT_SECRET": "******",
+        "AZURE_SSH_PUBLIC_KEY": "********"
+    }
+}
+```
+
+#### Defining Variable Overrides
+If you wish to override the default variables for flavor workers-- you can specify them as part of your tilt-settings.json as seen in the example below.  Please note-- precedence of variables is as follows:
+
+1. explicitly defined vars for each flavor i.e. worker-templates.flavors[0].AZURE_VNET_NAME
```

Comment:
will the instructions in https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/development.md#deploying-a-workload-cluster continue to work for using env vars to configure a workload cluster config if not using flavor as resources?

Reply:
Shouldn't be any interference between the two methods!  The flavors deployed via tilt will pull user-defined variables from tilt-settings.json (if present) whereas `make create-workload-cluster ` pulls them from environment variables